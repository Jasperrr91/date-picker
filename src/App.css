.day-picker {
  border: 1px solid #91918f;
  border-radius: 10px;
}