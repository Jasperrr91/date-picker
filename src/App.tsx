import React from 'react';
import './App.css';
import DayPicker from './components/DatePicker/DatePicker';

function App() {
  return (
    <div className="App">
      <DayPicker />
      <p>Just some underlying text that <br /> should be covered by the date picker</p>
    </div>
  );
}

export default App;
