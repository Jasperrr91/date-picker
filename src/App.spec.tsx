import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('Application', () => {
  it('It will render a Application component correctly', () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });
});
