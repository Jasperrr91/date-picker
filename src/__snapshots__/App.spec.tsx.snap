// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application It will render a Application component correctly 1`] = `
<div
  className="App"
>
  <DatePicker />
  <p>
    Just some underlying text that 
    <br />
     should be covered by the date picker
  </p>
</div>
`;
